### PR TITLE
Change CRT TV to LCD TV and Add Large LCD Screen Component

### PR DESCRIPTION
**Pull request by HarrisonGreenlee**
#### Summary
```SUMMARY: Content "Changes CRT TVs to LCD TVs and adds a large LCD screen component to the game that drops when the LCD TV is when disassembled."```

#### Purpose of change
CRT TVs are uncommon in the present day. Changing the TVs to LCD improves makes the game's setting feel more accurate.

#### Describe the solution
Change the default television's description, weight, and deconstruction recipe so that it appears to be a LCD TV instead of a CRT TV.

#### Describe alternatives you've considered
In the future I would like to add other types of televisions to the game, but for now this is a good starting point.

#### Testing
Found television, examined television to ensure it was changed as expected, and disassembled it. Picked up Large LCD Screen and examined it to ensure there are no issues with the component.

#### Additional context
N/A

### CODE REVIEW

**Comment by Xenomorph-III** [comment]
Add this to the console/broken console - odd that a computer has 2 small LCD screens.

File: data/json/items/generic.json [hunk 1]
```diff
@@ -1383,6 +1383,20 @@
...
   },
+  {
+    "type": "GENERIC",
+    "id": "large_lcd_screen",
+    "symbol": ",",
+    "color": "dark_gray",
+    "name": "large LCD screen",
+    "category": "spare_parts",
+    "description": "A large backlit screen, used for displaying images.  Useful in some electronics recipes.",
+    "price": 150000,
```
Comment (Night-Pryanik) [inline data/json/items/generic.json]: Please add a post-apocalypse price - I doubt these screens would cost $1500 after the Cataclysm.

Comment (HarrisonGreenlee) [reply data/json/items/generic.json]: [Price was far too high and has been changed to $100.](https://github.com/CleverRaven/Cataclysm-DDA/pull/35320/commits/33b0bbd29151b7523ab8a9f79db91456a4c939c8)

**Comment by HarrisonGreenlee** [comment]
@Xenomorph-III 
> Add this to the console/broken console - odd that a computer has 2 small LCD screens.


[Large LCD screen has been added to relevant crafting and deconstruction recipes as  suggested.](https://github.com/CleverRaven/Cataclysm-DDA/pull/35320/commits/33b0bbd29151b7523ab8a9f79db91456a4c939c8)  
The recipes should also provide some utility for large LCD screens, which will ensure that they are not a useless item.

File: data/json/furniture_and_terrain/terrain-manufactured.json [hunk 2]
```diff
@@ -640,6 +640,7 @@
         { "item": "RAM", "count": [ 4, 8 ] },
         { "item": "cable", "charges": [ 8, 16 ] },
         { "item": "small_lcd_screen", "count": [ 2, 4 ] },
+        { "item": "large_lcd_screen", "count": 1 },
```
Comment (Night-Pryanik) [inline data/json/furniture_and_terrain/terrain-manufactured.json]: 2..4 small LCD screens AND one large LCD screen, that's right?

File: data/json/items/generic.json [hunk 3]
```diff
@@ -1383,6 +1383,20 @@
...
+    "id": "large_lcd_screen",
+    "symbol": ",",
+    "color": "dark_gray",
+    "name": "large LCD screen",
+    "category": "spare_parts",
+    "description": "A large backlit screen, used for displaying images.  Useful in some electronics recipes.",
+    "price": 10000,
+    "material": "plastic",
+    "weight": "3000 g",
+    "volume": "1500 ml",
```
Comment (Night-Pryanik) [inline data/json/items/generic.json]: I don't know, isn't 1.5 liter volume too small for a "large" screen? What screen size do you have in mind?

File: data/json/furniture_and_terrain/terrain-manufactured.json (hunk 2 above)
Comment (HarrisonGreenlee) [reply data/json/furniture_and_terrain/terrain-manufactured.json]: In most cases I would agree that a device should only have a single screen, but this is a nuclear generator. I imagine it has a large screen and then a few smaller screens used for displaying data and status information. 
However if you disagree just let me know and I will remove the smaller screens.

File: data/json/items/generic.json (hunk 3 above)
Comment (HarrisonGreenlee) [reply data/json/items/generic.json]: Yeah, this is probably far too small for a "large screen".

In order to ensure I am getting a realistic value I will base the material and weight off of [this screen](http://www.panelook.com/LD550EUE-FHB1_LG%20Display_55_LCM_parameter_24253.html).

Weight: 13 kg = 13000 g
1232mm x 708.9mm x 17.0mm=14847 mL = approx. 15000 mL

I will change the JSON to fit this data.

File: data/json/furniture_and_terrain/terrain-manufactured.json (hunk 2 above)
Comment (Night-Pryanik) [reply data/json/furniture_and_terrain/terrain-manufactured.json]: That's OK.

**Pull request merged**